arch/arm/soc/st_stm32/stm32f0/Kconfig.defconfig.stm32f091xc

```diff
@@ -0,0 +1,22 @@
+# Kconfig - ST Microelectronics STM32F091XC MCU
```

Comment:
This should be in the SoC patch

Reply:
You could also make it generic to STM32F091XX  (rename to Kconfig.defconfig.stm32f091x**x**)

Reply:
Moved to soc patch. 

Did not make it generic. IMHO this should be done in a separate pull request.  I plan to do that when I adapt zephyr for my custom made F091xB board.